_includes/head.html

```diff
@@ -12,8 +12,8 @@
         <meta property="og:description" content="A consortium of universities, research labs, industry and developers producing open-source software and format standards for microscopy data." />
         <meta name="twitter:card" content="summary">
         <meta name="twitter:title" content="The Open Microscopy Environment">
-        <meta name="twitter:description" content="A consortium of universities, research labs, industry and developers producing open-source software and format standards for microscopy data.">
-        <meta name="twitter:image" content="http://www.openmicroscopy.org/img/logos/ome-logo_fb.png">        
+        <meta name="twitter:description" content="">
```

Comment:
I don't know what this will do left blank, probably better just to remove it, their dev docs say this isn't needed for the card to work